From 5dfaed08f51270ba495bdb3f7e9a287801efa953 Mon Sep 17 00:00:00 2001
From: Bhushan Shah <bhush94@gmail.com>
Date: Fri, 29 Jul 2016 17:11:05 +0530
Subject: [PATCH] debug

---
 src/main.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/main.cpp b/src/main.cpp
index afcfa49..f16046c 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -65,8 +65,8 @@ int main(int argc, char** argv)
     QString path;
     if (parser.isSet("phone") || qgetenv("PLASMA_PLATFORM") == QByteArray("phone")) {
         path = QStandardPaths::locate(QStandardPaths::DataLocation, "main.qml");
-        //qDebug() << "[DEBUG QStandardPaths::DataLocation] : " + QStandardPaths::DataLocation; 
-        //qDebug() << "[DEBUG path] : " + path;
+        qDebug() << "[DEBUG QStandardPaths::DataLocation] : " << QStandardPaths::DataLocation; 
+        qDebug() << "[DEBUG path] : " << path;
     }
     QQmlComponent component(&engine, path);
     if (component.isError()) {
-- 
2.9.0

